### PR TITLE
HPCC-24148 Move kubectl to platform-build

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith <jake.smith@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by jakesmith** [comment]
@richardkchapman - please review

**Comment by richardkchapman** [comment]
may need to change platform-build version?

**Comment by richardkchapman** [comment]
(or target 7.10)

**Comment by jakesmith** [comment]
@richardkchapman - have re-targeted to master/7.10

File: dockerfiles/incr.sh [hunk 1]
```diff
@@ -57,6 +57,8 @@ DOCKER_REPO=hpccsystems
 BUILD_TYPE=Debug
 [[ -n ${INPUT_BUILD_TYPE} ]] && BUILD_TYPE=${INPUT_BUILD_TYPE}
 BUILD_THREADS=$INPUT_BUILD_THREADS # If not set, picks up default based on nproc
+BUILD_USER=hpcc-systems
```
Comment (richardkchapman) [inline dockerfiles/incr.sh:60]: This change seems unrelated (and I'm not sure it's right - should instances of INPUT_BUILD_USER further down now say BUILD_USER? )

**Comment by richardkchapman** [comment]
Actually on closer inspection I'm not sure my concerns about the build version are correct - it's probably ok. There will be potential issues if you try to use incr on top of a base build that does not have the change in, but that's easily solved by rebasing the branch you are working on (and implies it was a bit out of date)

File: dockerfiles/incr.sh (hunk 1 above)
Comment (jakesmith) [reply dockerfiles/incr.sh:60]: yes it should. It is unrelated though.
I do need a way to specify the docker repo, so I can push incremental built images to my docker repo, and use them on Azure, GKE etc. - but will make mods in a separate PR.

**Comment by jakesmith** [comment]
@richardkchapman - removed unrelated change to incr.sh

File: dockerfiles/incr.sh (hunk 1 above)
Comment (jakesmith) [reply dockerfiles/incr.sh]: Correction: the [unrelated] change was about BUILD_USER not REPO. I need that so that when performing a -f (force) build, it picks up from my github user not hpccsystems.
(will deal with in separate PR).
